### ISSUE TITLE
Investigate Azure B2C for use with Terra

### ISSUE THREAD

**Issue by bvizzier-ucsc** [DataBiosphere/azul#5850]
Investigate Azure B2C and to review the document linked below and in the Epic description.

[The Terra team has provided a write-up with a discussion of the implementation](https://docs.google.com/document/d/1kvR4oIpo0P95dxlPjMLHhVp2vKEiMeCKvUWiw9l1iww/edit#heading=h.c6jqh9urbmex).

Megan Pare, Trevyn Langsford, and Tristian Garwood are part of the Terra team responsible for the B2C implementation in Terra available via the #ucsc-anvil-explorer-collab channel in Slack if there are questions.

**Comment by hannes-ucsc** [comment]
I've engaged with them in the document. I raised the following point

> Looking at the B2C documentation, I don't think it is feasible for my team to manage a separate B2C tenant. I see the following options: 1) the proxy solutions documented here (Azul talks OAuth to the proxy), 2) the Terra team makes the app registrations on the Azul teams behalf (Azul talks OAuth to Azure AD), 3) the Terra team grants restricted permissions to the Azul team to make their own app registrations for Terra's tenant. Even in option 1 there must be some form of app registration since how else would the proxy learn about valid redirect URLs. Note that we tend to manage many separate instances of Azul, often multiple personal instances for every developer. Ideally, the app registration would be automated.

They will disucuss internally.

TL;DR I don't think the B2C integration is baked enough for us to engage in an implementation at this point.

**Comment by bvizzier-ucsc** [comment]
Moving to icebox since Terra on Azure has been de-prioritized.
